### ISSUE TITLE
Course clone modal does not render in Firefox on Linux

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#542]
I tried in both Firefox and Iceweasel (on different Linux computers) and found that the course cloning feature is broken. (It still works on Chromium.)

@thenickcox @bmathews

**Comment by thenickcox** [comment]
@ragesoss Does this happen on master?

**Comment by bmathews** [comment]
Can't reproduce on ff on osx

**Comment by ragesoss** [comment]
I'll test on Windows. If it is Linux-only, we can ship before fixing it. It does happen on master. Here's my console explosion:

```
"[48]</CourseClonedModal<.render@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:143:2721
[430]</ReactCompositeComponentMixin._renderValidatedComponentWithoutOwnerOrContext@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:10059
[430]</ReactCompositeComponentMixin._renderValidatedComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:10246
[430]</ReactCompositeComponentMixin.mountComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:4854
[473]</ReactReconciler.mountComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1430:174
[430]</ReactCompositeComponentMixin._updateRenderedComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9751
[430]</ReactCompositeComponentMixin._performComponentUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9270
[430]</ReactCompositeComponentMixin.updateComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:8538
[430]</ReactCompositeComponentMixin.performUpdateIfNecessary@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:7816
[473]</ReactReconciler.performUpdateIfNecessary@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1430:706
runBatchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:1062
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:658
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:658
[483]</<.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:2834
[483]</flushBatchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:3148
[500]</Mixin.closeAll@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:1440
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:738
[445]</ReactDefaultBatchingStrategy.batchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1346:956
enqueueUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:1330
enqueueUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1457:40
[482]</ReactUpdateQueue.enqueueSetState@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1457:2821
[427]</ReactComponent.prototype.setState@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1292:863
[54]</Overview<.storeDidChange@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:161:766
Store/this.mixin.componentDidMount/this.listener<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:847:748
[121]</EventEmitter.prototype.emit@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:362:1361
[280]</Store.prototype.emitChange@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:847:1099
[106]</setWeeks@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:317:415
[106]</WeekStore<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:317:1335
[283]</Dispatcher</i.prototype._invokeCallback@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:859:1709
[283]</Dispatcher</i.prototype.dispatch@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:859:1524
[276]</Action.prototype.dispatch/</<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:835:481
C@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2520
D@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:3724
[276]</Action.prototype.dispatch/<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:835:350
P@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2265
x@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2351
S@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2168
f@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:779
" main-97b825abdc.js:835:82
reThrow/<() main-97b825abdc.js:835
d() raven.min.js:2
```

```
TypeError: r is undefined main-97b825abdc.js:143:2720
```

```
"[430]</ReactCompositeComponentMixin._updateRenderedComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9447
[430]</ReactCompositeComponentMixin._performComponentUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9270
[430]</ReactCompositeComponentMixin.updateComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:8538
[430]</ReactCompositeComponentMixin.receiveComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:7560
[473]</ReactReconciler.receiveComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1430:543
[430]</ReactCompositeComponentMixin._updateRenderedComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9537
[430]</ReactCompositeComponentMixin._performComponentUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:9270
[430]</ReactCompositeComponentMixin.updateComponent@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:8538
[430]</ReactCompositeComponentMixin.performUpdateIfNecessary@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1301:7816
[473]</ReactReconciler.performUpdateIfNecessary@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1430:706
runBatchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:1062
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:658
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:658
[483]</<.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:2834
[483]</flushBatchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:3148
[500]</Mixin.closeAll@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:1440
[500]</Mixin.perform@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1511:738
[445]</ReactDefaultBatchingStrategy.batchedUpdates@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1346:956
enqueueUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1460:1330
enqueueUpdate@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1457:40
[482]</ReactUpdateQueue.enqueueSetState@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1457:2821
[427]</ReactComponent.prototype.setState@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:1292:863
[54]</Overview<.storeDidChange@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:161:766
Store/this.mixin.componentDidMount/this.listener<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:847:748
[121]</EventEmitter.prototype.emit@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:362:1361
[280]</Store.prototype.emitChange@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:847:1099
[91]</setCourse@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:272:460
[91]</CourseStore<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:272:1403
[283]</Dispatcher</i.prototype._invokeCallback@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:859:1709
[283]</Dispatcher</i.prototype.dispatch@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:859:1524
[276]</Action.prototype.dispatch/</<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:835:481
C@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2520
D@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:3724
[276]</Action.prototype.dispatch/<@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:835:350
P@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2265
x@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2351
S@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:2168
f@https://dashboard.wikiedu.org/assets/javascripts/main-97b825abdc.js:851:779
"
```

```
TypeError: n is null main-97b825abdc.js:1301:9446
```

**Comment by ragesoss** [comment]
obviously, since it's on master, can't prevent shipping.

**Comment by ragesoss** [comment]
Appears to be a Firefox/Linux only bug. It works in FF on Windows, as well as Edge on Windows.

**Comment by bmathews** [comment]
@ragesoss I can't reproduce this locally, but I can reproduce it when on the deployed environments. Is that true for you? My immediate guess would be that this is a build/minification issue.

**Comment by bmathews** [comment]
Ah, it was a network race condition not reproducible locally until I throttled my connection. Fixed via 2cf30586b10756c2fc69bef5e1f5d8ca34606381
